crates/config/src/lib.rs

```diff
@@ -407,9 +404,8 @@ pub struct Config {
     /// CREATE2 salt to use for the library deployment in scripts.
     pub create2_library_salt: B256,
 
-    /// Compiler to use
-    #[serde(with = "from_str_lowercase")]
-    pub lang: Language,
+    /// Vyper optimization mode. "gas", "none" or "codesize"
+    pub vyper_optimize: Option<VyperOptimizationMode>,
```

Comment:
could we perhaps group this in a [vyper] table instead like

[vyper]
optimize = "gas"